backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceTestPerformedLoincCode.java

```diff
@@ -0,0 +1,26 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class DeviceTestPerformedLoincCode extends IdentifiedEntity {
+
+  @ManyToOne
+  @JoinColumn(name = "device_type_id")
+  private DeviceType deviceType;
```

Comment:
does it make sense for this to just be UUID instead if DeviceType type?

Reply:
I added `@JSONIgnore` to prevent json circular references when converting this object, but I don't think there is a lot of value to converting this to a UUID. Just to keep this ticket moving so we can begin backfilling data, I can create another one to resolve this issue that I can do shortly after this.

Reply:
thanks for adding @JSONIgnore per our convo on slack.